### ISSUE TITLE
[Feature] Ability to insert Unicode emoji from chat into the input field by clicking on them, just like emotes

### ISSUE THREAD

**Issue by RevSnowfox** [Xzensi/NipahTV#119]
Both emotes (native/7TV) and Unicode emoji serve similar purposes in chat; they add small images into text conversations, especially when rendered with Twemoji. While implementing emojis as an emote provider might be overkill, it makes sense to be able to "copy" the full set of little pictures seen in chat by clicking on them.

Of course the input field doesn't support Twemoji rendering (possible feature?), but the operating system should be able to display the Unicode characters with their native emoji font. (Or if not, having emojis as square boxes is preferable over not being able to insert them from chat.)

**Comment by Xzensi** [comment]
I agree it does make sense for the Emojis to be clickable the same as emotes, considering that emotes are clickable as well. There's some performance considerations due to the extra elements necessary forcing text nodes to be split up into even more elements to make them clickable, but I don't think it would pose an issue because on average there's many times more emotes than emojis anyway.
